log/core_wrapper.go

```diff
@@ -2,12 +2,25 @@ package log
 
 import (
 	"strconv"
+	"time"
 
 	"go.uber.org/zap/zapcore"
 
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
 	"github.com/reddit/baseplate.go/internal/thriftint"
 )
 
+var (
+	logLatencySeconds = promauto.NewHistogram(
```

Comment:
please fix the linter issue here: https://github.com/reddit/baseplate.go/actions/runs/13529151910/job/37806945821?pr=688
```suggestion
	logLatencySeconds = promauto.With(prometheusbpint.GlobalRegistry).NewHistogram(
```